src/osx/carbon/font.cpp

```diff
@@ -350,11 +366,22 @@ void wxFontRefData::MacFindFont()
                                 fontWithTraits = CTFontCreateCopyWithSymbolicTraits( m_ctFont, 0, remainingTransform, remainingTraits, remainingTraits );
                                 if ( fontWithTraits == NULL )
                                 {
-                                    // give in on the bold, try native oblique
+                                    // try native oblique, emulate bold later
                                     fontWithTraits = CTFontCreateCopyWithSymbolicTraits( m_ctFont, 0, NULL, kCTFontItalicTrait, kCTFontItalicTrait );
                                 }
+                                else
+                                {
+                                    remainingTraits &= ~kCTFontBoldTrait;
+                                }
                             }
                         }
+                        
+                        // we have to emulate bold
+                        if ( remainingTraits & kCTFontBoldTrait )
+                        {
+                            const float strokewidth = -3.0;
```

Comment:
It would be nice to have a comment saying where does this mysterious (to me, at least) `-3` come from.